modules/ossm-jaeger-config-elasticsearch.adoc

```diff
@@ -0,0 +1,77 @@
+////
+This TASK module included in the following assemblies:
+-assembly.adoc
+////
+
+[id="ossm-jaeger-config-elasticsearch_{context}"]
+= Configuring Elasticsearch for Jaeger
+
+The Jaeger collector requires persistent storage. Jaeger can be configured for different storage backends:
```

Comment:
I think the first sentence should be deleted - as is it not only the collector that requires persistent storage, it is also the query component - but in general it is better to just say "Jaeger can be configured...." - and then list the requirements for the different strategies as you have done.